test/JDBC/expected/drop_index-vu-verify.out

```diff
@@ -0,0 +1,532 @@
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: index "ix2t1_drop_indexbe63f9a1e6e197a9b4c58fbc7b470d87" does not exist)~~
+
+drop index if exists dbo.t1_drop_index.ix2
+go
+
+create index ix1 on dbo.t1_drop_index(a)
+go
+create index ix1 on guest.t1_drop_index(a)
+go
+select db_name(), object_name(id), object_schema_name(id), name from sysindexes where name like 'ix1%' order by 1,2,3,4
+go
+~~START~~
+nvarchar#!#varchar#!#varchar#!#varchar
+master#!#t1_drop_index#!#dbo#!#ix1t1_drop_indexa5111d2a1767bc43a700e9f2162be019
+master#!#t1_drop_index#!#guest#!#ix1t1_drop_indexa5111d2a1767bc43a700e9f2162be019
+~~END~~
+
+
+drop index t1_drop_index.ix1
+go
+select db_name(), object_name(id), object_schema_name(id), name from sysindexes where name like 'ix1%' order by 1,2,3,4
+go
+~~START~~
+nvarchar#!#varchar#!#varchar#!#varchar
+master#!#t1_drop_index#!#guest#!#ix1t1_drop_indexa5111d2a1767bc43a700e9f2162be019
+~~END~~
+
+drop index t1_drop_index.ix1
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: index "ix1t1_drop_indexa5111d2a1767bc43a700e9f2162be019" does not exist)~~
+
+drop index if exists t1_drop_index.ix1
+go
+
+-- dynamic SQL
+create index ix1 on dbo.t1_drop_index(a)
+go
+select db_name(), object_name(id), object_schema_name(id), name from sysindexes where name like 'ix1%' order by 1,2,3,4
+go
+~~START~~
+nvarchar#!#varchar#!#varchar#!#varchar
+master#!#t1_drop_index#!#dbo#!#ix1t1_drop_indexa5111d2a1767bc43a700e9f2162be019
+master#!#t1_drop_index#!#guest#!#ix1t1_drop_indexa5111d2a1767bc43a700e9f2162be019
+~~END~~
+
+execute('drop index guest.t1_drop_index.ix1')
+go
+select db_name(), object_name(id), object_schema_name(id), name from sysindexes where name like 'ix1%' order by 1,2,3,4
+go
+~~START~~
+nvarchar#!#varchar#!#varchar#!#varchar
+master#!#t1_drop_index#!#dbo#!#ix1t1_drop_indexa5111d2a1767bc43a700e9f2162be019
+~~END~~
+
+execute('drop index guest.t1_drop_index.ix1')
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: index "ix1t1_drop_indexa5111d2a1767bc43a700e9f2162be019" does not exist)~~
+
+execute('drop index if exists guest.t1_drop_index.ix1')
+go
+
+execute('drop index t1_drop_index.ix1')
+go
+select db_name(), object_name(id), object_schema_name(id), name from sysindexes where name like 'ix1%' order by 1,2,3,4
+go
+~~START~~
+nvarchar#!#varchar#!#varchar#!#varchar
+~~END~~
+
+execute('drop index t1_drop_index.ix1')
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: index "ix1t1_drop_indexa5111d2a1767bc43a700e9f2162be019" does not exist)~~
+
+execute('drop index if exists t1_drop_index.ix1')
+go
+
+-- stored proc
+create index ix1 on dbo.t1_drop_index(a)
+go
+create index ix1 on guest.t1_drop_index(a)
+go
+execute p1_drop_index
+go
+~~START~~
+nvarchar#!#varchar#!#varchar#!#varchar
+master#!#t1_drop_index#!#dbo#!#ix1t1_drop_indexa5111d2a1767bc43a700e9f2162be019
+~~END~~
+
+execute p1_drop_index
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: index "ix1t1_drop_indexa5111d2a1767bc43a700e9f2162be019" does not exist)~~
```

Comment:
In this error message, it would be more helpful to provide the actual index name the customer is trying to drop, such as "index 'ix1' does not exist." This can prevent confusion and help the customer identify the specific index causing the issue.

Reply:
True, but that is part of a much bigger issue, namely that hashed/mapped names should be hidden from the user and the original T-SQL names should be used instead. This would only be one case where this applies. Addressing this issue should be approached holistically rather than for this particular case.

Reply:
Understood. Thanks for the clarification!